### ISSUE TITLE
tube_sizing.py can't access tubing_data.txt, missing unit definitions

### ISSUE THREAD

**Issue by WPennock** [AguaClara-Reach/aguaclara#163]
I have been using tube_sizing.py with:
```python
from aguaclara.play import*
import aguaclara.research.tube_sizing as ts
```
Unless this file was intended to be used with another import statement, there appear to be a few problems.

The first is that when I use `ts.ID_colored_tube()`, it is unable to find `data/tubing_data.txt` where the definitions are. 

Second, I have to add this line at the beginning of my code
```python
u.define('rev = 1 * revolutions')
```
because there is mention of the `u.rev` unit in the code, but it does not appear to be defined. Without this line, `tube_sizing.py` cannot be imported.

**Comment by oliver-leung** [comment]
It looks like we'll have to fix the path to the `research/data/tubing_data.txt` file, and add that unit definition under `core/data/unit_definitions.txt`. @HannahSi , do you think you could put this change under v0.0.20?

**Comment by oliver-leung** [comment]
This issue has been resolved in #182 , and will be released with v0.0.21.
